### PR TITLE
feat(@kampus/sozluk-content): add tags as new terms

### PR DESCRIPTION
**Pull request by igdiaysu**
If the `tags` of the existing files were not available as files, they were created as `[sozluk-term].mdx` files. And tags were added to the new files.

### CODE REVIEW

File: packages/sozluk-content/terms/git.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title:
```
Comment (rasitds) [inline packages/sozluk-content/terms/git.mdx]: title is missing

File: packages/sozluk-content/terms/pipeline.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title:
```
Comment (rasitds) [inline packages/sozluk-content/terms/pipeline.mdx]: title is missing

File: packages/sozluk-content/terms/git.mdx (hunk 1 above)
Comment (igdiaysu) [reply packages/sozluk-content/terms/git.mdx]: resim

File: packages/sozluk-content/terms/pipeline.mdx (hunk 2 above)
Comment (igdiaysu) [reply packages/sozluk-content/terms/pipeline.mdx]: resim

File: packages/sozluk-content/terms/git.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Git
+tags:
+  - [[github]]
```
Comment (igdiaysu) [inline packages/sozluk-content/terms/git.mdx]:
```suggestion
  - git
  - [[github]]
```

**Comment by usirin** [comment]
Hello @igdiaysu 👋 TY so much for this work.

Did we have a discussion about putting wikilinks onto tags before? Tags are already easily parseable since it's in frontmatter part of the markdown, whereas wikilinks should ideally be used in the content itself.

I like having them linked through anyway though, so i don't think we should close this PR. But i think we can move them to the bottom of the text and list them as related content:

```markdown
---
title: something
tags:
- foo
- bar
---

## Bağlantılar
- [[related-sozluk-term|Related sozluk term]]
- [[related-sozluk-term2|Related sozluk term2]]
```

**Review by Ketcap — APPROVED** [review]

**Pull request merged**